packages/docs/content/12-enterprise-version/00-on-prem-system-requirements.md

```diff
@@ -0,0 +1,284 @@
+* SMTP Settings
+   - Configure SMTP settings to enable the Spectro Cloud platform to send out email notifications. Email Notifications are sent out to new users when they are on boarded to the platform so they can activate their accounts.
+* Certificate (optional)
+   - Provide the desired SSL or TLS server certificates to support external access to valid HTTPs.
+* FTP Location (backups)
+   - An accessible FTP location to backup data.
+   	
+# Quick Start Mode
+
+The Spectro Cloud On-Prem Quick Start Mode is a single node installation of the Spectro Cloud platform. This deployment is used to quickly understand the capabilities of the Spectro Cloud platform. For production purposes the enterprise version is recommended. The minimum requirements for Quick Start version is outlined below:
+
+### Hardware Requirements 
+
+Hardware requirements will describe the CPU, memory, and disk requirements for the nodes.
+* 8GB memory
+* 4 virtual CPUs 
+* Secondary storage of 80 GB distributed across multiple HDDs.
+
+### Network Requirements
+
+* Quick Start will operate on a single instance which requires connectivity through a proxy or non-proxy environment. 
+* This instance can be configured with static IP addresses or dynamically allocated IP addresses.
+
+<InfoBox>
+ Make sure that your Datacenter CIDR IP address does not overlap Kubernetes PodCIDR range. Kubernetes PodCIDR range settings can be changed while installing Quick Start Version
+</InfoBox>
+
+# Enterprise Mode
+
+The Spectro Cloud On-Premise Enterprise version is a highly available multinode infrastructure for production purposes. The minimum number of nodes/VMs required by this environment is 3. Depending on the number of nodes the requirements changes. The minimum requirements are outlined below:
+
+### Hardware Requirements 
+Hardware requirements will describe the CPU, memory, and disk requirements for the nodes.
+* Each VM requires 8GB memory resulting in 24GB RAM in total .
+* 4 vCPUs per node * 3 nodes requiring 12 vCPUs
+* Secondary storage of 240GB distributed across multiple HDDs.
+
+### Network Requirements
+
+* A Range or a Subnet of at least 5 static IP addresses is required for the installation and ongoing management. 
+* The infrastructure will require a proxy/non proxy environment for accessibility. 
+* For high availability purposes, you may choose to distribute the three VMs across multiple compute clusters.
+
+<InfoBox>
+ Make sure that your Datacenter CIDR IP address does not overlap Kubernetes PodCIDR range. Kubernetes PodCIDR range settings can be changed while installing Enterprise version
+</InfoBox>
+
+
+## Private Cloud Gateway Installer
+
+A Private Cloud Gateway is a Spectro Cloud component that enables the communication between Spectro Cloud's management console and a VMware based private data center. The gateway needs to be installed by the users in their VMware environments using a private cloud gateway installer appliance. 
```

Comment:
As this On prem we can mention:

A Private Cloud Gateway is a Spectro Cloud component that enables the communication between Spectro Cloud's management console and a data center which is not accessible directly via the Datacenter hosting Spectro Cloud On-Prem Application. The gateway needs to be installed by the users in their VMware environments using a private cloud gateway installer OVA. For more details <link to Gateway installer section>